src/quo2/components/inputs/title_input/view.cljs

```diff
@@ -0,0 +1,49 @@
+(ns quo2.components.inputs.title-input.view
+  (:require
+    [quo2.components.inputs.title-input.style :as style]
+    [reagent.core :as reagent]
+    [quo2.components.markdown.text :as text]
+    [react-native.core :as rn]))
+
+(defn- pad-0
+  [value]
+  (if (<= (count value) 1)
```

Comment:
im not sure this is needed, i guess its just in design for the demo purposes, there is an example `5/24` and there is no leading zero

Reply:
I think I meant `9`.  it should always be two digits. e.g `05/24` or `03/09` etc
Will check again to make sure it's right 👍

Reply:
yes, I will double check with design team 👍

Reply:
FYI: If we want to format with leading zeros, in CLJS we should probably use `format`:

```clojure
(goog.string/format "%02d" 9) ; => 09
```

Reply:
@ilmotta, this caused issues with the component tests. The compiler makes `goog` a global object so it doesn't play so nice with the tests. There's most likely a way to set this up, and something we will definitely need to address at some point. 
For now I left is as I would prefer to keep the tests on this and it's only some basic functionality but I will spend some time looking into how we can work with `goog` imports and component tests.

```
TypeError: goog.string.format is not a function
```

Reply:
No problem then, thanks for the info. It could be a good idea to create a quick issue if we haven't one.

Reply:
Yeah sure I can create an issue for this 👍